index.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
 
-var Requested = require('./requested')
-  , https = require('https')
-  , http = require('http');
+module.exports = (function () {
+  if ('XMLHttpRequest' in global) {
```

Comment:
What if we're testing browser-side code directly in `node`, e.g. via `jsdom`? The `require`'s will be resolved by `node` to just use `index.js`, but there is a global `XMLHttpRequest` we'd like to use that's exposed by `jsdom`. Or am I writing/running my tests incorrectly? Let me know the best practice here, thanks.